lib/internal/process/promises.js

```diff
@@ -5,6 +5,7 @@ const {
   ArrayPrototypeShift,
   Error,
   ObjectDefineProperty,
+  ObjectHasOwn,
```

Comment:
`Object.hasOwn` is not available on primordials on Node.js v16.x and v17.x because it can be disabled with a runtime flag (`--no-harmony-object-has-own`)
```suggestion
  ObjectPrototypeHasOwnProperty,
```

Reply:
Bummer, will do